### ISSUE TITLE
readline: build error with gcc 10

### ISSUE THREAD

**Issue by cproc** [genodelabs/genode#4103]
When building `LIB=readline` with gcc 10, the following error occurs:

```
  Library readline
    MERGE    readline.lib.so
genode-x86-ld: terminal.o:/.../contrib/readline-a0bfb07e4ee36edffd6eb908e2f57be2bb119fc8/src/lib/readline/terminal.c:103: multiple definition of `PC'; genode.o:/.../repos/libports/src/lib/readline/genode.cc:26: first defined here
genode-x86-ld: terminal.o:/.../contrib/readline-a0bfb07e4ee36edffd6eb908e2f57be2bb119fc8/src/lib/readline/terminal.c:103: multiple definition of `BC'; genode.o:/.../repos/libports/src/lib/readline/genode.cc:27: first defined here
genode-x86-ld: terminal.o:/.../contrib/readline-a0bfb07e4ee36edffd6eb908e2f57be2bb119fc8/src/lib/readline/terminal.c:103: multiple definition of `UP'; genode.o:/.../repos/libports/src/lib/readline/genode.cc:28: first defined here
```

**Comment by cproc** [comment]
Fixed in 43746a8

**Comment by chelmuth** [comment]
Merged to staging.
